README.md

```diff
@@ -1,33 +1,391 @@
-<p align="center">
-  <a href="https://www.clerk.com/?utm_source=github&utm_medium=starter_repos&utm_campaign=sdk_go" target="_blank" align="center">
-    <picture>
-      <source media="(prefers-color-scheme: dark)" srcset="./docs/clerk-logo-dark.png">
-      <img src="./docs/clerk-logo-light.png" height="64">
-    </picture>
-  </a>
-  <br />
-</p>
```

Comment:
🙃 You could probably use this logo:

https://github.com/clerk/javascript/blob/4fd4a90191217db0a2915f102748b0efcfeb2092/README.md?plain=1#L1-L9